Update with Experiment PHP local evaluation docs (#61)
amplitude/amplitude-dev-center: docs/experiment/sdks/php-sdk.md

    **Not getting the expected variant result for your flag?** Make sure your flag [is activated](../guides/getting-started/create-a-flag.md#activate-the-flag), has a [deployment set](../guides/getting-started/create-a-flag.md#add-a-deployment), and has [users allocated](../guides/getting-started/create-a-flag.md#configure-targeting-rules).

> ### Initialize Remote

Reviewer: ```suggestion
### Initialize remote evaluation
```

    }
}
```

## Local evaluation

> Implements evaluating variants for a user via [local evaluation](../general/evaluation/local-evaluation.md). If you plan on using local evaluation, you should [understand the tradeoffs](../general/evaluation/local-evaluation.md#targeting-capabilities).

Reviewer: ```suggestion
Implements evaluation of variants for a user through [local evaluation](../general/evaluation/local-evaluation.md). If you plan to use local evaluation, you should [understand the tradeoffs](../general/evaluation/local-evaluation.md#targeting-capabilities).
```

!!!note "Local Evaluation Mode"

>     The local evaluation client can only evaluation flags which are [set to local evaluation mode](../guides/create-local-evaluation-flag.md).

Reviewer: ```suggestion
    The local evaluation client can only evaluate flags which are [set to local evaluation mode](../guides/create-local-evaluation-flag.md).
```

!!!tip "Quick Start"

    1. [Initialize the local evaluation client.](#initialize-local)
    2. [Start the local evaluation client.](#start)
    3. [Evaluate a user.](#evaluate)

    ```php
    <?php
    // (1) Initialize the experiment client
    $experiment = new \AmplitudeExperiment\Experiment();
    $client = $experiment->initializeLocal('<DEPLOYMENT_KEY>');

    // (2) Start the local evaluation client.
    $client->start()->wait();

    // (3) Evaluate a user.
    $user = \AmplitudeExperiment\User::builder()
        ->deviceId('abcdefg')
        ->userId('user@company.com')
        ->userProperties(['premium' => true]) 
        ->build();

    // (3) Evaluate a user
    $variants = $client->evaluate($user);
    ```

    **Not getting the expected variant result for your flag?** Make sure your flag [is activated](../guides/getting-started/create-a-flag.md#activate-the-flag), has a [deployment set](../guides/getting-started/create-a-flag.md#add-a-deployment), and has [users allocated](../guides/getting-started/create-a-flag.md#configure-targeting-rules).

> ### Initialize Local

Reviewer: ```suggestion
### Initialize local evaluation
```

> Initializes a [local evaluation](../general/evaluation/local-evaluation.md) client.

Reviewer: ```suggestion
For more information, see [Local Evaluation](../general/evaluation/local-evaluation.md).
```

!!!warning "Server Deployment Key"

>     You must [initialize](#initialize-local) the local evaluation client with a server [deployment](../general/data-model.md#deployments) key to get access to local evaluation flag configs.

Reviewer: ```suggestion
    [Initialize](#initialize-local) the local evaluation client with a server [deployment](../general/data-model.md#deployments) key to access local evaluation flag configurations.
```

```php
initializeLocal(string $apiKey, ?LocalEvaluationConfig $config = null): LocalEvaluationClient
```

| Parameter | Requirement | Description |
| --- | --- | --- |

> | `apiKey` | required | The server [deployment key](../general/data-model.md#deployments) which authorizes fetch requests and determines which flags should be evaluated for the user. |

Reviewer: ```suggestion
| `apiKey` | required | The server [deployment key](../general/data-model.md#deployments) which authorizes fetch requests and determines which flags to evaluate for the user. |
```

| `config` | optional | The client [configuration](#configuration_1) used to customize SDK client behavior. |

#### Configuration

You can configure the SDK client on initialization.

???config "Configuration Options"

    | <div class="big-column">Name</div> | Description | Default Value |
    | --- | --- | --- |
    | `debug` | Set to `true` to enable debug logging. | `false` |
    | `serverUrl` | The host to fetch flag configurations from. | `https://api.lab.amplitude.com` |
    | `bootstrap` | Bootstrap the client with an array of flag key to flag configuration | `[]` |

!!!info "EU Data Center"

>     If you're using Amplitude's EU data center, configure the `serverUrl` option on initialization to `https://api.lab.eu.amplitude.com`

Reviewer: ```suggestion
    If you use Amplitude's EU data center, configure the `serverUrl` option on initialization to `https://api.lab.eu.amplitude.com`
```

### Start

Fetch local evaluation mode flag configs for [evaluation](#evaluate).

```php
start(): PromiseInterface
```

> You should await the result of `start()` to ensure that flag configs are ready to be used before calling [`evaluate()`](#evaluate)

Reviewer: ```suggestion
Await the result of `start()` to ensure that flag configs are ready for use before you call [`evaluate()`](#evaluate)
```

```php
<?php
$client->start()->wait();
```

### Evaluate

> Executes the [evaluation logic](../general/evaluation/implementation.md) using the flags fetched on [`start()`](#start). You must give evaluate a user object argument. You can optionally pass an array of flag keys if you require only a specific subset of required flag variants.

Reviewer: ```suggestion
Executes the [evaluation logic](../general/evaluation/implementation.md) using the flags fetched on [`start()`](#start). Give `evaluate()` a user object argument. Optionally pass an array of flag keys if you require only a specific subset of required flag variants.
```

```php
evaluate(User $user, array $flagKeys = []): array
```

| Parameter | Requirement | Description |
| --- | --- | --- |
| `user` | required | The [user](../general/data-model.md#users) to evaluate. |

> | `flagKeys` | optional | Specific flags or experiments to evaluate. If empty, all flags and experiments are evaluated. |

Reviewer: ```suggestion
| `flagKeys` | optional | Specific flags or experiments to evaluate. If empty, Amplitude evaluates all flags and experiments. |
```
